contracts/RedstoneOracleDemo.sol

```diff
@@ -0,0 +1,32 @@
+/// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity 0.8.4;
+
+import "@redstone-finance/evm-connector/contracts/data-services/MainDemoConsumerBase.sol";
+import {Ownable} from '@openzeppelin8/contracts/access/Ownable.sol';
+
+contract RedstoneOracleDemo is MainDemoConsumerBase, Ownable {
```

Comment:
We use 2-step ownership transfer as implemented in the previous [`contracts/RedstoneOracle.sol`](https://github.com/GammaStrategies/hypervisor/blob/98d83ef10eec0f907a6064d2b48313d84ced39a9/contracts/RedstoneOracle.sol) contract.

Reply:
Thanks for noticing that. Could we use the https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable2Step.sol  (is it consistent with your approach)? 
I'd prefer to derive from audited librry, just in case.

Reply:
yea, okay.